azure-pipelines.yml

```diff
@@ -44,8 +44,7 @@ jobs:
     buildJobName: Build_Windows_Debug
     testArtifactName: Transport_Artifacts_Windows_Debug
     configuration: Debug
-    testArguments: -testDesktop -test32
-    queueName: 'BuildPool.Windows.10.Amd64.ES.VS2017.Open'
+    testArguments: -testDesktop -test32 -helixQueueName Windows.10.Amd64.Server19H1.ES.Open
```

Comment:
There will be some hiccups when we migrate to Helix, complex system stuff will go wrong. I feel like it's invetible we'll hit a few periods where we want to flip away from Helix, go back to the old system while we work out the kinks and then flip back. 

Ideally there would just be a nice global variable we flip in the YML to achieve this. That though feels like it will make the scripts really awkward: double the queue information, lot more `if` in them to change how it works. Definitely would be hard to follow and a bit more up front work. 

At the same time the only other option for "temporarily disable Helix" is revert this change. This change is more than just flipping our infra to Helix, it's also changing how run tests work, changing how our scripts work, etc ... That could be done independently because they're all designed to work on Helix or without Helix. Wonder if we should do the following: 

1. Get this change to a point where we all agree and approve it. 
1. Do not commit this change 
1. Open two new PRs: one PR that contains everything but the YML changes and one that contains only the YML changes. Merge them in that order. 

Then if we do hit a bug where we need to temporarily disable Helix we would just be reverting the YML. That's probably more managable than all the other work that is coming along here. 

Thoughts? Basically typing while I'm thinking on this one.

Reply:
I think that to start with, a two-PR solution makes plenty of sense. If we want to disable helix in the short term then reverting the second PR will be a good way to do it. I also think that in the medium term we might end up wanting to revert helix on specific jobs, etc., or just "turn off" helix test runs after a number of intermediate changes have been merged into this file, at which point we'd probably want to create a "helix" pipeline variable.

Reply:
Note that *some* yaml changes will be needed to make CI work even with helix disabled. For example, we might need to introduce a command that searches for and runs all the `rehydrate.cmd/sh` scripts, or write out a top-level script that runs all of those scripts.